cmd/testworkflow-toolkit/artifacts/junit_post_processor.go

```diff
@@ -63,10 +62,7 @@ func (p *JUnitPostProcessor) Add(path string) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to read %s", path)
 	}
-	ok, err := isJUnitReport(xmlData)
-	if err != nil {
-		return errors.Wrapf(err, "failed to check if %s is a JUnit report", path)
-	}
+	ok := isJUnitReport(xmlData)
```

Comment:
Probably best to not load the whole `xmlData` then - just the first 8KB as below